Source/Meadow.Foundation.Core/Sensors/Buttons/PushButton.cs

```diff
@@ -67,32 +60,33 @@ public bool State
         protected IDigitalInputPort DigitalIn { get; set; }
 
         /// <summary>
-        /// Raised when a press starts (the button is pushed down - circuit is closed)
+        /// Raised when a press starts
         /// </summary>
         public event EventHandler PressStarted = delegate { };
 
         /// <summary>
-        /// Raised when a press ends (the button is released - circuit is opened)
+        /// Raised when a press ends
         /// </summary>
         public event EventHandler PressEnded = delegate { };
 
         /// <summary>
-        /// Raised when the button circuit is re-opened after it has been closed (at the end of a press)
+        /// Raised when the button is released after a press
         /// </summary>
         public event EventHandler Clicked = delegate { };
 
         /// <summary>
-        /// Raised when the button is pressed for LongClickedThreshold or longer
+        /// Raised when the button is pressed for LongClickedThreshold or longer and then releases
```

Comment:
how about "Raised when the button is released after being pressed for LongClickedThreshold or longer"?